pkg/repo/repo.go

```diff
@@ -47,15 +51,16 @@ func (repo *Repo) LoadMessages(id string) error {
 	err := repo.db.View(func(txn *badger.Txn) error {
 		item, err := txn.Get([]byte(id))
 		if err != nil {
-			return err
+			return fmt.Errorf("getting key/value pair: %w", err)
```

Comment:
Also should be error checks otherwise lint is failing

Reply:
Here everything is okay